tests/test_util.py

```diff
@@ -222,12 +222,19 @@ async def asyncSetUp(self):
     async def asyncTearDown(self):
         await self.http.close()
 
+    async def test_correct_content_type(self):
+        await get_extra_data_info_from_url(
+            url=f"https://ftp.gnu.org/gnu/gzip/gzip-1.12.tar.gz",
+            session=self.http,
+            content_type_deny=[re.compile(f"^{self._CONTENT_TYPE}$")],
+        )
+
     async def test_wrong_content_type(self):
-        with self.assertRaises(CheckerError):
+        with self.assertRaises(CheckerFetchError):
             await get_extra_data_info_from_url(
-                url=f"https://httpbin.org/response-headers?Content-Type={self._CONTENT_TYPE}",
+                url=f"https://httpbingo.org/response-headers?Content-Type=application/gzip",
                 session=self.http,
-                content_type_deny=[re.compile(f"^{self._CONTENT_TYPE}$")],
+                content_type_deny=[re.compile(f"^application/json$")],
```

Comment:
Shouldn't be an f-string.

```suggestion
                content_type_deny=[re.compile(r"^application/json$")],
```